src/dup/grouped.py

```diff
@@ -3,9 +3,15 @@
 
 
 def same_bytes(left_name, right_name):
-    left_bytes = open(left_name, "rb").read()
-    right_bytes = open(right_name, "rb").read()
-    return left_bytes == right_bytes
+  left_file = open(left_name, "rb")
```

Comment:
thanks for the PR - can you please adjust to 4-space indentation? Happy to merge after that.

Reply:
what you mean sir can you explain please?